contrib/prometheus/consoles/synapse.html

```diff
@@ -159,29 +159,28 @@ <h3>Database scheduling latency</h3>
 <script>
 new PromConsole.Graph({
   node: document.querySelector("#synapse_storage_schedule_time"),
-  expr: "rate(synapse_storage_schedule_time:total[2m]) / 1000",
-  name: "Total latency",
+  expr: "rate(synapse_storage_schedule_time_sum[2m]) / rate(synapse_storage_schedule_time_count[2m])",
```

Comment:
we should probably fix the description of this graph to say "/Average/ database scheduling latency" ?